lib/persisted-model.js

```diff
@@ -654,6 +676,22 @@ module.exports = function(registry) {
 
     setRemoting(PersistedModel, 'replaceOrCreate', replaceOrCreateOptions);
 
+    setRemoting(PersistedModel, 'upsertWithWhere', {
+      aliases: ['patchOrCreateWithWhere'],
+      description: 'Update an existing model instance or ' +
+      'insert a new one into the data source based on the ' +
+      'where criteria.',
```

Comment:
@superkhau I did run the eslint before recommitting and it dint report me any errors. I have changed the indentation now. Pls check if that's fine.